jest-preset.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
   snapshotFormat: {
     printBasicPrototype: false,
   },
+  workerThreads: true,
```

Comment:
all the tests passed for me even when turning this off -> maybe it isn't needed anymore?  I've seen this problem before too but usually it was when I was doing `expect(1).toBe(1n)` or something.  When the types match, it seems to work fine.  Take another look please, but if not please add your comment in the code so future readers will know why it's needed.

```
  workerThreads: true, // Enabled so jest would stop complaining about serializing BigInt.  See https://github.com/jestjs/jest/issues/11617#issuecomment-1458155552 for details
```

Reply:
It only passes because there are no errors in the tests. If you disable this flag and then cause a comparison that fails with a bigint, it'll fail again. Adding a comment makes sense since it is far from obvious.

Reply:
```suggestion
  workerThreads: true, // Enabled so jest would stop complaining about serializing BigInt.  See https://github.com/jestjs/jest/issues/11617#issuecomment-1458155552 for details
```